blog/2025-07-cluster-api-provider-gardener.md

```diff
@@ -0,0 +1,129 @@
+
+<!-- truncate -->
+
+### What is cluster-api-provider-gardener (CAPGa)?
+
+CAPGa allows you to manage Kubernetes clusters using declarative `cluster.x-k8s.io` Cluster API manifests, with Gardener acting as the cloud-independent provider and cluster orchestrator.
+
+Specifically, CAPGa implements the Cluster API’s provider interfaces to manage Gardener’s [Shoot clusters](https://gardener.cloud/about/) as Cluster API `Cluster` resources. This enables users to provision, update, and delete clusters managed by Gardener via standard Cluster API tooling and workflows.
+
+The following figure depicts the semantic mapping of Custer API (CAPI) resources to Gardener API (GAPI) resources:
+
+![Image showing the interaction of CAPI and Gardener through CAPGa](./static/capi-interaction-gardener-capga.svg)
+
+### What is the difference between Gardener API and Cluster API?
+
+Gardener and Cluster API both aim to automate Kubernetes cluster management, but they approach the problem from opposite directions and with different user audiences in mind.
+
+#### Gardener API: Platform-first and top-down
+
+Gardener is a top-down architected platform, designed to deliver homogeneous Kubernetes clusters across clouds as a Service. It was built with application or service teams in mind who require consistent, production-ready Kubernetes clusters without managing internal details.
+- A single manifest defines an entire cluster (`Shoot`) with platform-chosen defaults.
+- Users benefit from a user experience layer, sensible defaults, and well-integrated extensions (e.g. DNS, worker pools, cloud-specific settings).
+- Cluster lifecycle is abstracted away: versioning, networking, and OS images are handled centrally.
+- Clusters can be managed for multiple cloud providers simultaneously.
+
+This setup is depicted in the following illustration, where the Gardener API is used to manage clusters as a service:
+
+![Illustration: Gardener managed by service team](./static/gardener-managed-by-service-team.svg)
+
+#### Cluster API: Infrastructure-centric and bottom-up
+
+Cluster API (CAPI), on the other hand, is a bottom-up framework for building cluster management solutions. It’s aimed at infrastructure teams who need fine-grained control over every component of a Kubernetes cluster and know how to assemble them.
+- A cluster is defined via multiple resource manifests (Cluster, MachineDeployment, KubeadmConfig, etc.).
+- Each resource is reconciled by dedicated controllers.
+- There’s no opinionated user interface. Users must understand the internals of Kubernetes cluster bootstrapping.
+- Powerful for building your own platform, but requires substantial operational ownership.
+
+This setup is illustrated below, where the end user manages the full lifecycle of the Cluster API management plane:
+
+![Illustration: Cluster API self-managed](./static/cluster-api-self-managed.svg)
```

Comment:
arrow to k8s cluster must come from capi box.
also, graphically, name the box "various capi-provider/controller" box, with state.